salt/metalk8s/metalk8s-ui/files/metalk8s-ui_deployment.yaml

```diff
@@ -0,0 +1,54 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    k8s-app: ui
+    kubernetes.io/name: "MetalK8sUI"
```

Comment:
Same as earlier comment, can be part of same tracking/debt ticket.

Reply:
Debt ticket created : #835